lib/multi_factor_auth.rb

```diff
@@ -102,4 +102,10 @@ def self.domestic_country_code(phone_number)
     end
     false
   end
+
+  def self.choose_mfa_method(user)
+    raise Disabled unless is_enabled?
```

Comment:
I see we've used this above so not really a think of this PR but i wonder if we could rename this to `MFADisabled`?

Reply:
In #671 I actually went for removing the enable/disable MFA feature flag entirely, because having different behaviour in prod and dev isn't great

Reply:
riight, ace ok i'll resolve this then.
Cheers for the link